package/MDAnalysis/analysis/diffusionmap.py

```diff
@@ -364,10 +365,191 @@ def transform(self, n_eigenvectors, time):
         time : float
             Exponent that eigenvalues are raised to for embedding, for large
             values, more dominant eigenvectors determine diffusion distance.
-        Return
-        ------
+
+
+        Returns
+        -------
         diffusion_space : array (n_frames, n_eigenvectors)
             The diffusion map embedding as defined by [Ferguson1]_.
         """
-        return (self._eigenvectors[1:n_eigenvectors+1,].T *
+        return (self._eigenvectors[1:n_eigenvectors+1, ].T *
                 (self.eigenvalues[1:n_eigenvectors+1]**time))
+
+    def plot_animated_transform(self, n_time=25, colorscale='Viridis', marker_size=3,
+                                x=1, y=2):
+        """ Plots an interactive 2D plot of chosen eigenvectors at different timescales.
+
+        Parameters
+        ----------
+        n_time: int, optional
+            number of time values for ``transform()``. Default: 25
+        colorscale: str, optional
+            Plotly colorscale for coloring. Default: 'Viridis'
+        marker_size: float, optional
+            Marker size for plot. Default: 3
+        x: int, optional
+            Eigenvector to plot on x-axis. Default: 1
+        y: int, optional
+            Eigenvector to plot on y-axis. Default: 2
+
+        Returns
+        -------
+        plotly.graph_objs._figure.Figure
+
+
+        .. versionadded:: 1.0.0
+        """
+        try:
+            import plotly.graph_objects as go
+        except ImportError:
+            raise ImportError('Plotly is required for this animated graph')
+
+        if n_time is None:
```

Comment:
`n_frames` instead of `n_time` cold be a clearer name?

Reply:
I've changed it to `n_frames` for now because it *does* pass integers representing frames to ``transform()``, but I'm not sure that doesn't make it more confusing given that the keyword for ``transform`` is ``time``. What do you think of docstring/s?